core/src/main/java/nl/nn/adapterframework/util/StringResolver.java

```diff
@@ -50,7 +50,9 @@ public static String getSystemProperty(String key, String def) {
         try {
             return System.getProperty(key, def);
         } catch (Throwable e) { // MS-Java throws com.ms.security.SecurityExceptionEx
-            log.warn("Was not allowed to read system property [" + key + "]: "+ e.getMessage());
+        	System.out.println("Was not allowed to read system property [" + key + "]: "+ e.getMessage()); // somehow StringResolver use in LogUtil is conflicting, therefor switched to System.out.println()
+            //log.warn("Was not allowed to read system property [" + key + "]: "+ e.getMessage());
+        	
```

Comment:
Dat komt omdat de logger dan nog niet geïnstalleerd is. Ik stel voor alle referenties naar de logger we te halen (ook de import)